src/buttons/src/IconButton.js

```diff
@@ -83,8 +88,16 @@ class IconButton extends PureComponent {
   }
 
   render() {
-    const { theme, iconAim, icon, height, intent, ...props } = this.props
-    const iconSize = theme.getIconSizeForIconButton(height)
+    const {
+      theme,
+      iconAim,
+      icon,
+      iconSize,
+      height,
+      intent,
+      ...props
+    } = this.props
+    const size = iconSize || this.props.theme.getIconSizeForIconButton(height)
```

Comment:
`this.props.theme.getIconSizeForIconButton(height)` => `theme.getIconSizeForIconButton(height)` since you already deconstruct it!